sdk/communication/azure-communication-identity/mypy.ini

```diff
@@ -0,0 +1,13 @@
+[mypy]
+python_version = 3.10
```

Comment:
We should test the minimum supported python for the SDK == 3.7

Reply:
Done